### DISCUSSION TITLE
Drag and Drop items does not work

### DISCUSSION THREAD

**Discussion by danieleavitabile** [wezterm/wezterm#6379] [Q&A]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

In MacOs, dropping a file from the file browser into wezterm results in its full path being pasted. I expected something similar should happen with other applications too. 

For instance, I use [Zotero](https://www.zotero.org) to manage bibliographies for my research. In iTerm I can open a text file using, say Neovim, and drag one item of the library to Neovim. This results in the corresponding BibTex entry, that is, a long piece of text with tags associated to the bibliography, to be copied into the text file.

For some reason this type of behaviour does not work in Wezterm: when I drag the item on the text file nothing happens. Any pointers towards solving this problem? 

I attach a screen recording where you can see the behaviour in iTerm vs wezterm (they are using the exact same Neovim and zsh setup)

https://github.com/user-attachments/assets/878ca089-903c-45c6-874d-8155de43c3f4

**Comment by yuri-urban** [comment]
I have the same kinda of issue with dragging Apple Mail links onto WezTerm. In Ghostty it just works, but WezTerm ignores such drag-n-drops. Couldn't find a way to enable it...
